elf2flt.c

```diff
@@ -1948,7 +1951,14 @@ int main(int argc, char *argv[])
 	  | (pic_with_got ? FLAT_FLAG_GOTPIC : 0)
 	  | (docompress ? (docompress == 2 ? FLAT_FLAG_GZDATA : FLAT_FLAG_GZIP) : 0)
 	  );
-  hdr.build_date = htonl((uint32_t)time(NULL));
+
+  sde = getenv("SOURCE_DATE_EPOCH");
+  if (sde)
+      t = (time_t) strtoul(sde, NULL, 10);
```

Comment:
no space between the cast & func call

is it even needed though ?  let the compiler figure it out ?